src/actions/breakpoints/index.js

```diff
@@ -179,9 +176,7 @@ export function toggleBreakpoints(
  */
 export function removeAllBreakpoints() {
   return async ({ dispatch, getState }: ThunkArgs) => {
-    const breakpointList = getBreakpoints(getState())
-      .valueSeq()
-      .toJS();
+    const breakpointList = (Object.values(getBreakpoints(getState())): any);
```

Comment:
I think it would be nice to have a selector `getBreakpointList` that returns an array of breakpoints and can be flow typed